benchmarks/benchmarks-core/src/jmh/java/io/micrometer/benchmark/BenchmarkSupport.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2025 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.benchmark;
+
+import io.micrometer.common.lang.Nullable;
```

Comment:
Please rebase on the latest `main` and update any nullability annotations to use the jSpecify annotations.